src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,51 @@
+package model;
+
+public class FruitTransaction {
+    private static final String CAN_NOT_FIND_OPERATION = "Can not find suitable Operation ";
+    private final Operation operation;
+    private String fruit;
+    private final int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public enum Operation {
```

Comment:
As far as you are using this enum in several places, what do you think about separate file for that?